Makefile

```diff
@@ -54,6 +60,11 @@ test-workspace:
 		--exclude hdk_derive \
 		--locked \
 		--features $(A)
+	$(F) RUST_BACKTRACE=1 cargo nextest run \
```

Comment:
Run tests against wasmer_sys and wasmer_wamr feature flags.

I spent a while trying to modify the nix flake scripts for running tests until I realized it didn't seem like they're being used? In which case the core_testing.md doc is outdated?